meshstack-hub/.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
f: Please add this file to .gitignore.

Reply:
![image](https://github.com/user-attachments/assets/d3401a94-33ea-42b9-8365-b7a25fa0528b)
It should already o.o

Reply:
will delete them manually